keyboards/rgbkb/zen/rev1/keymaps/starcaller_v1.1/keymap.c

```diff
@@ -0,0 +1,152 @@
+#include QMK_KEYBOARD_H
+
+uint16_t copy_paste_timer;
+
+enum custom_keycodes {
+  S_H = SAFE_RANGE, // slack here
+  S_H_P,       // slack here + paste 
+  T_H_B,         // ticket header bike
+  T_H_T,       // ticket header tread 
+  E_OP,  	    // email open
+  E_CL,		    // email close 
+  E_FU,		    // email follow up 
+  E_SS,       // email service schedule
+  SCRN_C,	    // screen clip 
+  DC_C,       // double click + copy
+  KC_CCCV,    // one key copy/paste 
+  		
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+
+    case S_H:
+      if (record->event.pressed) {
+        SEND_STRING("@HERE");
+      }
+ break;
+
+    case S_H_P:
+      if (record->event.pressed) {
+        SEND_STRING("@HERE" SS_TAP(X_SPC) SS_LCTL("v"));
+      }
+ break;
+
+ case T_H_B:
+      if (record->event.pressed) {
+        SEND_STRING("Your Peloton Bike - ");
+      }
+ break;
+
+ case T_H_T:
+      if (record->event.pressed) {
+        SEND_STRING("Your Peloton Tread - ");
+      }
+ break;
+
+    case E_OP:
+      if (record->event.pressed) {
+        // when keycode E_OP is pressed
+        SEND_STRING("Hi , "SS_TAP(X_ENTER)SS_TAP(X_ENTER)"Thank you for contacting Peloton."SS_TAP(X_ENTER)SS_TAP(X_ENTER));
+      }
+break;
+
+    case E_CL:
+      if (record->event.pressed) {
+                SEND_STRING("Please let me know if you have any other questions."SS_TAP(X_ENTER)SS_TAP(X_ENTER)"Thank you for being the best part of Peloton."); 
+      }
+break;
+
+    case E_FU:
+      if (record->event.pressed) {
+                SEND_STRING("I will be personally watching this issue and will reach out to you after this is complete to make sure everything went smoothly."SS_TAP(X_ENTER)SS_TAP(X_ENTER)); 
+      }
+break;
+
+    case E_SS:
+      if (record->event.pressed) {
+                SEND_STRING("Please click here to schedule your service."SS_TAP(X_ENTER)SS_TAP(X_ENTER)); 
+      }
+break;
+
+    case SCRN_C:
+      if (record->event.pressed) {
+                tap_code16(C(S(KC_F5))); 
+      }
+break;
+
+   case DC_C:
+      if (record->event.pressed) {
+                tap_code(KC_BTN1);
+                tap_code(KC_BTN1);
+                tap_code16(C(KC_C)); 
+      }
+break;
+    
+    case KC_CCCV:  // One key copy/paste
+            if (record->event.pressed) {
+                copy_paste_timer = timer_read();
+            } else {
+                if (timer_elapsed(copy_paste_timer) > TAPPING_TERM) {  // Hold, copy
+                    register_code(KC_LCTL);
+                    tap_code(KC_C);
+                    unregister_code(KC_LCTL);
+                } else {  // Tap, paste
+                    register_code(KC_LCTL);
+                    tap_code(KC_V);
+                    unregister_code(KC_LCTL);
```

Comment:
This can be replaced with just `tap_code16(C(KC_C))`

Reply:
revised